src/codec/string.rs

```diff
@@ -71,4 +69,15 @@ mod test {
 
         Ok(())
     }
+
+    #[test]
+    fn test_string_codec2() -> Result<()> {
```

Comment:
i see what you did there, sneakily upping the coverage :P

Reply:
Ja there was no new uncovered line :( so I had to be creative. string had the lowest coverage I could find so 🗡️ that test!